### PR TITLE
Fix marathon (~6K sites), CSVFeedSpider fun

### PR DESCRIPTION
**Pull request by mjoe999**

### CODE REVIEW

File: locations/spiders/marathon.py [hunk 1]
```diff
@@ -1,39 +1,23 @@
...
+        "country": "US",
+    }
+    start_urls = [
+        "https://www.marathonbrand.com/content/includes/mpc-brand-stations/SiteList.csv"
+    ]
+
+    def parse_row(self, response, row):
+        if row["Status"] == "Open":
+            row["street_address"] = row["Address"]
+            del row["Address"]
```
Comment (iandees) [inline locations/spiders/marathon.py:20]:
```suggestion
            row["street_address"] = row.pop("Address")
```

**Comment by iandees** [comment]
I wonder if it needs a different User-Agent?

**Comment by mjoe999** [comment]
How annoying. It was working for me as it was using a local cache of the page from months ago.

**Comment by Cj-Malone** [comment]
I just came across this, the [file](https://www.marathonbrand.com/content/includes/mpc-brand-stations/SiteList.csv) is now base64'ed but it also isn't accessible via curl or scrapy...

**Pull request merged**